app/services/selection.js

```diff
@@ -200,11 +206,24 @@ export default Service.extend({
       const inSelection = selected.features.find(selectedFeature => selectedFeature.properties.geoid === properties.geoid);
 
       if (inSelection === undefined) {
-        selected.features.push({
-          type,
-          geometry,
-          properties,
-        });
+
+        if (this.get('summaryLevel') === 'boroughs' || this.get('summaryLevel') === 'cdtas') {
```

Comment:
Are we sure we only want to use this fix for boroughs and cdtas? I could imagine leaving out tracks and blocks but wouldn't is also be a noticeably issue with NTAs and CDs?

Reply:
Good question. I checked again and districts and cities benefits from this fix as well. I'll add those. However I can't replicate the issue with NTAs